tests/flows/test_sklearn.py

```diff
@@ -0,0 +1,483 @@
+                  "  svd_solver='auto', tol=0.0, whiten=False\)\)\)\)."
+        self.assertRaisesRegexp(ValueError, fixture, sklearn_to_flow, pipeline)
+
+        fu = sklearn.pipeline.FeatureUnion((('pca1', pca), ('pca2', pca2)))
+        fixture = "Found a second occurence of component sklearn.decomposition.pca.PCA when trying to serialize " \
+                  "FeatureUnion\(n_jobs=1,\n" \
+                  "       transformer_list=\(\('pca1', PCA\(copy=True, " \
+                  "iterated_power='auto'," \
+                  " n_components=None, random_state=None,\n" \
+                  "  svd_solver='auto', tol=0.0, whiten=False\)\), \('pca2', " \
+                  "PCA\(copy=True, iterated_power='auto'," \
+                  " n_components=None, random_state=None,\n" \
+                  "  svd_solver='auto', tol=0.0, whiten=False\)\)\),\n" \
+                  "       transformer_weights=None\)."
+        self.assertRaisesRegexp(ValueError, fixture, sklearn_to_flow, fu)
+
+        fs = sklearn.feature_selection.SelectKBest()
+        fu2 = sklearn.pipeline.FeatureUnion((('pca1', pca), ('fs', fs)))
+        pipeline2 = sklearn.pipeline.Pipeline((('fu', fu2), ('pca2', pca2)))
+        fixture = "Found a second occurence of component " \
+                  "sklearn.decomposition.pca.PCA when trying to serialize " \
+                  "Pipeline\(steps=\(\('fu', FeatureUnion\(n_jobs=1,\n" \
+                  "       transformer_list=\(\('pca1', PCA\(copy=True, " \
+                  "iterated_power='auto'," \
+                  " n_components=None, random_state=None,\n"  \
+                  "  svd_solver='auto', tol=0.0, whiten=False\)\), " \
+                  "\('fs', SelectKBest\(k=10, score_func=<function " \
+                  "f_classif at 0x[a-z0-9]+>\)\)\),\n" \
+                  "       transformer_weights=None\)\), \('pca2', " \
+                  "PCA\(copy=True, iterated_power='auto'," \
+                  " n_components=None, random_state=None,\n" \
+                  "  svd_solver='auto', tol=0.0, whiten=False\)\)\)\)."
+        self.assertRaisesRegexp(ValueError, fixture, sklearn_to_flow, pipeline2)
+
+    def test_subflow_version_propagated(self):
+        this_directory = os.path.dirname(os.path.abspath(__file__))
+        sys.path.append(this_directory)
+        import dummy_learn.dummy_forest
+        pca = sklearn.decomposition.PCA()
+        dummy = dummy_learn.dummy_forest.DummyRegressor()
+        pipeline = sklearn.pipeline.Pipeline((('pca', pca), ('dummy', dummy)))
+        flow = sklearn_to_flow(pipeline)
+        # In python2.7, the unit tests work differently on travis-ci; therefore,
+        # I put the alternative travis-ci answer here as well. While it has a
+        # different value, it is still correct as it is a propagation of the
+        # subclasses' module name
+        self.assertIn(flow.external_version,
+                      ['dummy_learn==1.0,sklearn==0.18.1',
```

Comment:
You're still hard-coding the sklearn version here.... you can import it and get it that way?